src/data/monsters.txt

```diff
@@ -1351,6 +1351,17 @@ The Big Ignatowicz	2218	firethedude.gif	BOSS NOCOPY Atk: 250 Def: 225 HP: 2000 I
 The Man on Fire	2219	firetheman.gif	BOSS NOCOPY Atk: 250 Def: 225 HP: 2000 Init: 60 P: dude E: hot	really dense meat stack (100)
 The Naughty Scorcheress	2220	sorcform1.gif	BOSS NOCOPY Atk: 190 Def: 171 HP: 400 Init: 10000 P: dude ED: hot Article: The	Instant Karma (c100)	Thwaitgold maggot statuette (n100)
 
+# Fall of the Dinosaurs
+
```

Comment:
According to [Tokoeka](https://kolmafia.us/members/tokoeka.4856/)

2230 - kachungasaur, dino_kachung.gif